### PR TITLE
PWGHF: fix of the histogram registry in the Lc task

### PR DESCRIPTION
**Pull request by DelloStritto**
- Update of the histogram registry in the Lc task in agreement with the pull request: https://github.com/AliceO2Group/AliceO2/pull/5050/files

- Fixing histograms and binning in the Lc task accordingly to the AliPhysics task.

### CODE REVIEW

File: Analysis/Tasks/PWGHF/taskLc.cxx [hunk 1]
```diff
@@ -70,29 +70,29 @@ struct TaskLc {
...
+      registry.get<TH1>(HIST("hCPA"))->Fill(candidate.cpa());
+      registry.get<TH1>(HIST("hEta"))->Fill(candidate.eta());
+      registry.get<TH1>(HIST("hselectionstatus"))->Fill(candidate.isSelLcpKpi());
+      registry.get<TH1>(HIST("hselectionstatus"))->Fill(candidate.isSelLcpiKp());
+      registry.get<TH1>(HIST("hImpParErr"))->Fill(candidate.errorImpactParameter0());
+      registry.get<TH1>(HIST("hImpParErr"))->Fill(candidate.errorImpactParameter1());
+      registry.get<TH1>(HIST("hImpParErr"))->Fill(candidate.errorImpactParameter2());
+      registry.get<TH1>(HIST("hDecLenErr"))->Fill(candidate.errorDecayLength());
+      registry.get<TH1>(HIST("hDecLenErr"))->Fill(candidate.chi2PCA());
     }
```
Comment (jgrosseo) [inline Analysis/Tasks/PWGHF/taskLc.cxx:94]: You should use registry.fill instead...

**Review by jgrosseo — CHANGES_REQUESTED** [review]

File: Analysis/Tasks/PWGHF/taskLc.cxx (hunk 1 above)
Comment (DelloStritto) [reply Analysis/Tasks/PWGHF/taskLc.cxx:94]: Hi @jgrosseo. I'm propagating the updates from the D0task that you can find here:

https://github.com/AliceO2Group/AliceO2/pull/5050/files#diff-7da89d2b691b60a9b20ec58616b3e46d2ba476b280c559bdc9dd4df035d83a39

Anyway, If we decide for a different strategy we have to propagate it to all the task.

Comment (jgrosseo) [reply Analysis/Tasks/PWGHF/taskLc.cxx:94]: Yes, please use fill and not get unless you keep the pointer to the histogram. It should be straightforward to replace it. Let me know if you run into any trouble.

Comment (DelloStritto) [reply Analysis/Tasks/PWGHF/taskLc.cxx:94]: Hi @jgrosseo . I modified the task following your indication. Let me know if it looks good to you. If so, I can take care of the propagation to the other tasks and push them before that the pull request is merged.
Thanks a lot for the review!

**Review by jgrosseo — DISMISSED** [review]

File: Analysis/Tasks/PWGHF/taskLc.cxx (hunk 1 above)
Comment (jgrosseo) [reply Analysis/Tasks/PWGHF/taskLc.cxx:94]: Looks good. Thanks.

**Review by jgrosseo — DISMISSED** [review]

**Review by vkucera — CHANGES_REQUESTED** [review]
Please fix the tab in front of `continue;}*/` on line 67 in `taskLc.cxx` (and other tabs if any).

**Review by vkucera — APPROVED** [review]
Thanks. Looks good to me.

**Review by ginnocen — APPROVED** [review]

**Pull request merged**